homework.js

```diff
@@ -0,0 +1,207 @@
+//Q1
+
+const  summation = function (n) {
+  let output = 0;
+  for( i = 1;  i <= n; i++) {
+    output = output  + i;
+  }
+  return output ;
+};
+
+console.log('Sum of numebers from 1 to '+ 5 +' is '+ summation(5));
+
+
+//Q2
+
+
+const largestEven = function (arr) {
+  const evens = [];
+
+  for (i =1; i < arr; i++) {
+    if(i % 2 === 0) {
+      evens.push(i);
+    }
+
+  }
+ 
+ function myFunc(total, num) {
+  return total + num;
+}
+  console.log(evens.reduce(myFunc))
+ 
+  }    
+  console.log(largestEven(5));
```

Comment:
Not readable, don't know how to test it. please have a one function that sums all the even numbers equal or less than the input number.